http.go

```diff
@@ -293,6 +297,40 @@ func (resp *Response) BodyWriter() io.Writer {
 	return &resp.w
 }
 
+// BodyStream returns io.Reader
+//
+// You must close it after you use it.
+func (resp *Response) BodyStream() io.ReadCloser {
+	if resp.bodyStream == nil {
+		resp.bodyStream = bytes.NewReader(resp.Body())
+	}
+	return newCloseReader(resp.bodyStream, resp.closeBodyStream)
+}
```

Comment:
Is this method useful? We could also not add it for now.

Reply:
We hope to read the response body through stream to prevent excessive memory pressure